central-portal/src/main/kotlin/com/vanniktech/maven/publish/portal/SonatypeCentralPortal.kt

```diff
@@ -12,19 +12,18 @@ import retrofit2.converter.scalars.ScalarsConverterFactory
 
 public class SonatypeCentralPortal(
   private val baseUrl: String,
-  private val usertoken: String,
+  private val userToken: String,
   userAgentName: String,
   userAgentVersion: String,
-  okhttpTimeoutSeconds: Long,
-  private val closeTimeoutSeconds: Long,
+  okhttpTimeout: Duration,
```

Comment:
This is a breaking change, but since no users are expected to rely on it, it's safe to land.

Reply:
Reverted the usage of `Duration` due to #1144.